### PR TITLE
Add Data Storage deprecations section

### PR DESCRIPTION
**Pull request by ishan1729**
This change needs to be reviewed first by one of our team member as well.

### CODE REVIEW

**Comment by Adam-Dupaski** [comment]
Hi Ishan, who is going to review this?

**Comment by ishan1729** [comment]
Hello Adam, Reviewing task will be assigned to someone during the standup at 10. I will communicate on this thread.

**Comment by ishan1729** [comment]
Adam - @nerush is going to review this change first.

File: content/refguide/moving-from-6-to-7.md [hunk 1]
```diff
@@ -291,3 +291,26 @@ This is needed to avoid potential namespace conflicts between the Mendix version
 #### 6.3.7 Runtime Issues When Migrating a Project to Mendix 7
 
 Java libraries in Mendix 7 shipped with the installation package are not available for projects anymore. While this results in better dependency management for each project, it can also cause errors at runtime after migration (for example, `NoClassDefFoundError`). Therefore, it is important to make sure the `userlib` directory of the migrated project includes all the required libraries. It is also worth noting that in Mendix 7, only one version of each library can exist at runtime. This means that if there are multiple versions of one library, the latest version is used and the rest are ignored.
+
+### 6.4 Removed Data Storage Functionality
+#### Removed Methods
+
+Package Name | Method Name | Alternative |
```
Comment (nerush) [inline content/refguide/moving-from-6-to-7.md]: Prepend **Package Name** column with pipe.

**Pull request merged**